benchmark/run.rb

```diff
@@ -1,15 +1,40 @@
-if RUBY_ENGINE == 'opal'
-  require 'opal/compiler'
-  require 'nodejs'
-  require 'corelib/string/unpack'
+require 'open3'
+require 'opal/os'
+OS = Opal::OS
+
+files = ARGV
+
+# ruby exes
+opal_chrome_exe = OS.windows? ? 'bin/opal-bench-chrome.bat' : 'bin/opal-bench-chrome'
+opal_firefox_exe = OS.windows? ? 'bin/opal-bench-firefox.bat' : 'bin/opal-bench-firefox'
+opal_node_exe = OS.windows? ? 'bin/opal-bench-node.bat' : 'bin/opal-bench-node'
+ruby_exe = Gem.ruby
+
+# rubies
+rubies = ''
+
+if ARGV.delete('-node')
```

Comment:
opal now supports optparse, perhaps it would be a good idea to rewrite it in terms of it?

Reply:
done